engine/table/src/main/java/io/deephaven/engine/table/impl/ImmutableColumnSource.java

```diff
@@ -57,4 +61,9 @@ default short getPrevShort(final long elementIndex) {
     default boolean isImmutable() {
         return true;
     }
+
+    @Override
+    default void fillPrevChunk(@NotNull FillContext context, @NotNull WritableChunk<? super Values> destination, @NotNull RowSequence rowSequence) {
```

Comment:
Great idea. Let's do it for `getPrevChunk`, too. Be sure to check that nothing is overriding any of these methods without a good reason.

I wonder if this actually does anything, since `AbstractColumnSource` is the parent for most implementations.